appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -0,0 +1,344 @@
+package nu.yona.server
+
+import groovyx.net.http.RESTClient
+import spock.lang.Ignore
+import spock.lang.IgnoreRest
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+import groovy.json.*
+
+import javax.mail.*
+import javax.mail.search.*
+
+class CreateUserOnBuddyRequestTest extends Specification {
+
+	def adminServiceBaseURL = System.properties.'yona.adminservice.url'
+	def YonaServer adminService = new YonaServer(adminServiceBaseURL)
+
+	def analysisServiceBaseURL = System.properties.'yona.analysisservice.url'
+	def YonaServer analysisService = new YonaServer(analysisServiceBaseURL)
+
+	def appServiceBaseURL = System.properties.'yona.appservice.url'
+	def YonaServer appService = new YonaServer(appServiceBaseURL)
+	@Shared
+	def timestamp = YonaServer.getTimeStamp()
+
+	@Shared
+	def richardQuinPassword = "R i c h a r d"
+	def bobDunnPassword = "B o b"
+	@Shared
+	def richardQuinURL
+	@Shared
+	def richardQuinLoginID
+	@Shared
+	def bobDunnGmail = "bobdunn325@gmail.com"
+	@Shared
+	def bobDunnGmailPassword = "bobbydunn"
```

Comment:
Someone will find this password and hijack the account. Let's add a configuration property that indicates whether we're running production or test. In case of test, return the URL in the buddy DTO.